packages/DropdownMenu/stories/examples/DropdownMenuExample.js

```diff
@@ -2,8 +2,11 @@ import React from "react";
 import Confirmation from "@paprika/confirmation";
 import DropdownMenu from "../../src";
 
-const handleConfirm = onCloseMenu => onCloseConfirm => {
-  onCloseConfirm();
+const handleConfirm = onConfirm => {
```

Comment:
Updated the examples to show correct implementation of the confirmation component, where we close the menu after the confirmation panel has been animated closed.